src/settings.py

```diff
@@ -170,7 +170,6 @@ class GlobalSettings:
     side_predicate_learner = "no_learning"
 
     # torch model parameters
-    normalization_scale_clip = 1
```

Comment:
isn't removing this a functional change?

Reply:
well in the place where it's used, it's a kwarg that defaults to 1. but you're right that technically this changes some (unused) functionality... gonna leave it though